README.md

```diff
@@ -94,7 +95,20 @@ A lot. Various things that are missing include:
 
 ## Install
 
-Currently the only method of install is from source control.
+The sambacc library, samba-container command, and samba-dc-container
+assume an it is being run within an OCI container. While there's
```

Comment:
Something is not right with the above sentence 😕.

Reply:
Fixed